duckdb-dataframe/benchmark.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Install
+
+sudo apt-get update
+sudo apt-get install -y python3-pip
+pip install pandas chdb
```

Comment:
I think you want `pip install pandas duckdb pyarrow` here? The latter is the backend for loading parquet files.